content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md

```diff
@@ -194,7 +194,7 @@ const instance = new os.compute.Instance("test", {
 exports.instanceIP = instance.accessIpV4;
 ```
 
-Check out the [API documentation]({{< relref "/docs/reference/pkg/nodejs/pulumi/openstack" >}})
+Check out the [Resource documentation]({{< relref "docs/intro/cloud-providers/openstack/" >}})
```

Comment:
In this context, I think we should link to `/docs/reference/pkg/openstack/` rather than the intro page?

But again whichever one we choose, that is, either linking to the resource docs page under `/docs/reference/pkg...` or `docs/intro/cloud-providers/...` we should be consistent throughout this page.

Reply:
Agreed this is good change, but I think it may make sense to leave the two links pictured below as is, let me know if you think otherwise: 
![image](https://user-images.githubusercontent.com/41454626/79268105-1353da00-7e4f-11ea-839f-a4bdba045c7c.png)

Reply:
I ended up using the /docs/reference/pkg links consistently through page, as you suggested.